src/components/DynamicCalendar/eventCal/DayView.js

```diff
@@ -198,13 +198,15 @@ export default class DayView extends React.PureComponent {
   }
 
   render() {
-    const { styles } = this.props
+    const { styles, height, marginTop } = this.props
+    const PACKED_EVENTS_CONTAINER_HEIGHT = height -  styles?.header?.height - marginTop
```

Comment:
In the case where `styles?.header?.height` is undefined should we do something like `(styles?.header?.height || 0)` to avoid the possibility of `PACKED_EVENTS_CONTAINER_HEIGHT` ending up being NaN?

Reply:
good point!